app/Models/CustomTour.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class CustomTour extends Model
+{
+    protected $connection = 'tours';
```

Comment:
Add conditional environment test as in `database/migrations/2023_09_25_113832_create_tours_table.php`

Reply:
Thanks, I've added this